src/execution.rs

```diff
@@ -396,9 +396,10 @@ impl Scan {
             for (index, arm) in self.arms.iter().enumerate() {
                 let captures = arm.regex.captures(&match_string[i..]);
                 if let Some(captures) = captures {
-                    if !captures.get(0).unwrap().range().is_empty() {
-                        matches.push((captures, index));
+                    if captures.get(0).unwrap().range().is_empty() {
+                        panic!("Empty match for regular expression /{}/", arm.regex);
```

Comment:
Using `/` as the delimiters seems a bit nonstandard, given that the syntax used in an actual piece of code doesn't use these delimiters. 

As long as it's unambiguous, though, I think it's fine.

Reply:
It was already used in the format string of an existing exception. It is maybe even nicer if this was used as the syntax as well, instead of quotes, to be more explicit about it being a regex instead of a simple string.

Reply:
Good point!